Can_Node_Handler.cpp

```diff
@@ -156,3 +167,27 @@ void Can_Node_Handler::writeThrottleMessages(const int16_t throttle) {
   };
   CAN().write(rightFrame);
 }
+void Can_Node_Handler::writeBrakeMessages(const int16_t brake){
```

Comment:
Why is this function necessary? Could you just send a negative value to writeThrottleMessages instead?